### ISSUE TITLE
Fight system enhancement

### ISSUE THREAD

**Issue by truepaddii** [REGoth-project/REGoth#302]
I've got a request that stated it would be nice if there was free aiming for melee _and_ ranged combat so that the system is more skill based. Also, the handling of the hero character movement and the camera should be enhanced so that there is a lower response time (faster response in an interface – also input/output – is essential for a smooth feeling). These options could be optional if players demand the old mechanic but REGoth focuses on the strengths of Gothic and the clumsy movement and controls, in general, were weaknesses.
You may contact me for clarification or further information on this feature request.

**Comment by Jurigag** [comment]
Exactly, at least free aiming for ranged combat should be implemented beacause there are already mods for this for original G1 and G2. Also for now it could be nice to add some hit detection if we attack one enemy if other should be hit too, then add free aiming for melee maybe.

**Comment by DietrichDaniel** [comment]
Agreed. That enhancement would allow to disable enemy scanning by using bow. It would be great if the enemy name would appear only if we pointed at it. Free aim with meele also sounds great. It's frustrating when you can hit only enemy the player has focus on. But it would also raise problems, like accidentaly hitting NPCs, so my proposal is to make it work only on aggressive enemies.
